benchmarks/Sentry.Benchmarks/ProfilingBenchmarks.cs

```diff
@@ -125,9 +139,9 @@ public void DiagnosticsSessionStartCopyStop(bool rundown, string provider)
     {
         EventPipeProvider[] providers = provider switch
         {
-            "runtime" => new[] { new EventPipeProvider("Microsoft-Windows-DotNETRuntime", EventLevel.Informational, (long)ClrTraceEventParser.Keywords.Default) },
+            "runtime" => new[] { new EventPipeProvider("Microsoft-Windows-DotNETRuntime", EventLevel.Informational, (long)BenchmarkDotNetTransientTraceEvent::Microsoft.Diagnostics.Tracing.Parsers.ClrTraceEventParser.Keywords.Default) },
```

Comment:
We'll need to fix this in a follow up PR. Since our customers will experience the same thing. We need to change namespaces, or make stuff internal

Reply:
related to https://github.com/getsentry/sentry-dotnet/pull/2385#discussion_r1301976435